amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -133,6 +133,14 @@ extern  "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_Resizetensor(vx_graph
                                      RPP VX_API_ENTRY C Function NODE TENSOR
 *************************************************************************************************************/
 extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_Brightness(vx_graph graph, vx_tensor pSrc, vx_tensor srcROI, vx_tensor pDst, vx_array alpha, vx_array beta, vx_scalar input_layout, vx_scalar output_layout, vx_scalar roiType);
+extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_GammaCorrection(vx_graph graph, vx_tensor pSrc, vx_tensor srcROI, vx_tensor pDst, vx_array alpha, vx_scalar input_layout, vx_scalar output_layout, vx_scalar roiType);
```

Comment:
Change input_layout and output_layout to camel case to maintain uniformity

Change in Brightness also